### DISCUSSION TITLE
Export html and CCS to use with any project

### DISCUSSION THREAD

**Discussion by seaj1one** [thewebforge/structeezy-docs#2] [Ideas]
My idea is to be able to build components or patterns have can export html, CCS and JS into any Buildrius but this could apply to any project. 

Even though the main focus is Bricks but with this feature, you can still focus on Bricks. I'm able to prototype using Bricks and still benefit, In other builders, such as Builderius, Pinegrow or any builder that has the ability.

**Comment by cbontems** [comment]
Hello,

Can't you do this by reusing the html / css / js you get on Bricks front-end after prototyping in Bricks ?
What would be the benefit of using Structeezy for that ?

Thanks!

**Reply by seaj1one** [reply]
Hello Cédric 

I never knew you could do that. I just tried and the html seems to be pretty easy but the CSS I seemed to copy lots of CSS when I targeted a section. 

- One of the things that I was thinking is removing any reference to bricks. 
- Also eliminating human error in case someone doesn't copy all or the wrong code. 
- Any exports that you provide will just be well thought out and correct.
